content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -236,78 +285,154 @@ versions the API server knows how to send, attempts to call to the webhook will
 This example shows the data contained in an `AdmissionReview` object
 for a request to update the `scale` subresource of an `apps/v1` `Deployment`:
 
-```json
+
+{{< tabs name="AdmissionReview_request" >}}
+{{% tab name="admission.k8s.io/v1" %}}
+```yaml
```

Comment:
Is the switch to yaml here intentional? Better for the comments?

Reply:
Yeah, it renders the comments in a nicer way

Reply:
JSON doesn't allow comments, but they're fine in YAML.